test/util.spec.js

```diff
@@ -44,17 +44,31 @@ describe('util', () => {
     })
   })
 
-  it('.serialize and .deserialize large objects', (done) => {
+  describe('.serialize and .deserialize large objects', () => {
     const size = 65536 * 2
     const largeObj = { someKey: [].slice.call(new Uint8Array(size)) }
+    let serialized
 
-    dagCBOR.util.serialize(largeObj, (err, serialized) => {
-      expect(err).to.not.exist()
-      expect(Buffer.isBuffer(serialized)).to.equal(true)
+    it('should grow to accomodate large objects', (done) => {
+      dagCBOR.util.serialize(largeObj, (err, _serialized) => {
+        serialized = _serialized
```

Comment:
FYI: I tend to use a leading underscore to use it for parameters that aren't used in the code (this is how you suppress a warning about it in e.g. Erlang or Rust).

Also having tests depend on other tests is often a source of problems, I'd probably just serialize the node again in the test below.

Reply:
👍 solves the eslint nested callback complaints too